docs/source/Coding/Setting-up-PyCharm.md

```diff
@@ -1,57 +1,104 @@
 # Setting up PyCharm with Evennia
 
-[PyCharm](https://www.jetbrains.com/pycharm/) is a Python developer's IDE from Jetbrains available for Windows, Mac and Linux. It is a commercial product but offer free trials, a scaled-down community edition and also generous licenses for OSS projects like Evennia.
+[PyCharm](https://www.jetbrains.com/pycharm/) is a Python developer's IDE from Jetbrains available for Windows, Mac and Linux. 
+It is a commercial product but offer free trials, a scaled-down community edition and also generous licenses for OSS projects like Evennia.
 
-> This page was originally tested on Windows (so use Windows-style path examples), but should work the same for all platforms.
+First, download and install the IDE edition of your choosing.
+The community edition should have everything you need, 
+but the professional edition has integrated support for Django which can help.
 
-First, install Evennia on your local machine with [[Getting Started]]. If you're new to PyCharm, loading your project is as easy as selecting the `Open` option when PyCharm starts, and browsing to your game folder (the one created with `evennia --init`). We refer to it as `mygame` here. 
+## From an existing project
 
-If you want to be able to examine evennia's core code or the scripts inside your virtualenv, you'll
-need to add them to your project too:
+First, ensure you have completed the steps outlined [here](https://www.evennia.com/docs/latest/Setup/Installation.html#requirements)
+Especially the virtualenv part, this will make setting the IDE up much easier.
 
-1. Go to `File > Open...`
-1. Select the folder (i.e. the `evennia` root)
-1. Select "Open in current window" and "Add to currently opened projects"
+1. Open Pycharm and click on the open button, open your root folder corresponding to `mygame/`.
+2. Click on File -> Settings -> Project -> Python Interpreter -> Add Interpreter -> Add Local Interpreter
+![Example](https://imgur.com/QRo8O1C.png)
+3. Click on VirtualEnv -> Existing Interpreter -> Select your existing `evenv` folder and click ok
```

Comment:
Just be clear, clarify that you want the virtualenv folder (`evenv` if you followed the default installation)